Dockerfile.tasks

```diff
@@ -1,43 +1,44 @@
-FROM node:20.18.1 as build
-ENV NEXT_TELEMETRY_DISABLED 1
+FROM node:22.15.1 AS build
+
+ENV NEXT_TELEMETRY_DISABLED=1
 ARG WS="@helpwave/tasks"
 
-# install pnpm
-RUN npm install --global pnpm@9
+RUN npm install -g pnpm@9
 
 WORKDIR /web
 
-# copy all dependency declarations inside the container
-COPY package.json .
-COPY pnpm-lock.yaml .
-COPY pnpm-workspace.yaml .
+COPY package.json pnpm-lock.yaml pnpm-workspace.yaml ./
 COPY tasks/package.json ./tasks/
 COPY api-services/package.json ./api-services/
 
-# install dependencies
 RUN CI=1 pnpm install
 
-# build tasks app
 COPY tasks ./tasks/
 COPY api-services ./api-services/
+
 RUN pnpm --filter $WS run build
 
-FROM node:20.18.1-alpine
+
+FROM node:22.15.1-alpine
 
 LABEL maintainer="tech@helpwave.de"
 
-ENV NEXT_TELEMETRY_DISABLED 1
+ENV NEXT_TELEMETRY_DISABLED=1
+
 WORKDIR /web
 
-RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
+RUN addgroup -g 1001 nodejs && \
+    adduser -S -u 1001 -G nodejs nextjs
 
-# copy transpiled app
 COPY --from=build --chown=nextjs:nodejs /web/tasks/build/standalone ./
 COPY --from=build --chown=nextjs:nodejs /web/tasks/public ./tasks/public
 COPY --from=build --chown=nextjs:nodejs /web/tasks/build/static ./tasks/build/static
 
+USER nextjs
+
 EXPOSE 80
-ENV PORT 80
-ENV HOSTNAME 0.0.0.0
+ENV PORT=80
```

Comment:
nextjs is not root, as such one needs to manually set `PORT` when deploying to a >1024 port. you should probably fix that